src/sentry/ownership/grammar.py

```diff
@@ -20,9 +20,11 @@
 
 rule = _ matcher owners
 
-matcher      = _ matcher_tag identifier
+matcher      = _ matcher_tag any_identifier
 matcher_tag  = (matcher_type sep)?
-matcher_type = "url" / "path"
+matcher_type = "url" / "path" / event_tag
+
+event_tag   = ~r"tag.[^:]+"
```

Comment:
In search queries we use `tags[tagname]` but I don't love that syntax, I would like to one day add `tags.tagname` to search, should we use `tags.` here too? It uses the same inflection as the property on events and how we refer to tags elsewhere in the product.

Reply:
i would prefer the dot syntax to become the standard because we also have this with `frame.filename` and stuff like this.

Reply:
Changed it to `tags.tagname` now.

Reply:
any particular reason to go with `tags` vs the singular `tag`? latter seems more consistent with path/url/frame.filename etc.?